pkg/asset/installconfig/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package installconfig generates the install config assets based on its dependencies
```

Comment:
nit: you're missing a trailing period here.

And maybe add a note about these being the main settings we expect users to edit?